Common/Data/Custom/Tiingo/TiingoNewsJsonConverter.cs

```diff
@@ -118,8 +118,13 @@ public static TiingoNews DeserializeNews(JToken token)
             var symbols = new List<Symbol>();
             foreach (var tiingoTicker in tickers)
             {
-                var ticker = TiingoSymbolMapper.GetLeanTicker(tiingoTicker.ToString());
-
+                var rawTicker = tiingoTicker.ToString();
+                if (rawTicker.Contains(" ") || rawTicker.Contains("|"))
+                {
+                    continue;
```

Comment:
Suggest we add a trace/error log with the raw ticker being ignored.
Wonder how many cases are there of this and if it would require a custom ticker converter/mapper